crates/CrateAssembler.kt

```diff
@@ -52,20 +50,20 @@ class CrateAssembler : Callable<Unit> {
 
     @Option(names = ["--deps"])
     lateinit var deps: String
-    private val depsList: Array<String>
-        get() = if (deps.isEmpty()) emptyArray() else deps.split(";").toTypedArray()
 
     @Option(names = ["--dep-features"])
     lateinit var depFeatures: String
     private val depFeaturesList: Array<String>
         get() = if (depFeatures.isEmpty()) emptyArray() else depFeatures.split(";").toTypedArray()
 
+    @Option(names = ["--dep-workspaces"])
+    lateinit var depWorkspaces: String
```

Comment:
To properly convert external bazel workspace dependencies into Crate dependencies, we must pass workspace refs.